HumanDynamicsEstimationLibrary/utils/src/iDynTreeUtils.cpp

```diff
@@ -391,8 +391,15 @@ void state::Integrator::integrate(const iDynTree::VectorDynSize& new_dot_s, cons
 void state::Integrator::integrate(const iDynTree::VectorDynSize& new_dot_s,
                                   const iDynTree::Vector3& new_dot_W_p_B,
                                   const iDynTree::Vector3& new_omega_B,
-                                  const double dt)
+                                  const double dt,
+                                  bool* resetFlag)
 {
+    if(*resetFlag) {
+        *resetFlag = false;
+        std::cout << "Reset old state integrator" << std::endl;
```

Comment:
I don't know if we want to keep this print.

Reply:
I would like to leave it. I don't know if we can make it like a yarp debug print

Reply:
@dariosortino @davidegorbani can you reach a consensus on this, perhaps aligning in person? Thanks!

Reply:
If you want to leave, I would suggest to add something more descriptive to the print, otherwise just seeing a line "Reset old state integrator"  in a terminal is not really descriptive, I would add "HDE iDynTree: state::Integrator::integrate: " to the message to make it more clear.

Reply:
I removed the print in https://github.com/robotology/human-dynamics-estimation/pull/388/commits/8521b425355be1196298bf533f13f14915b6dfda